templates.go

```diff
@@ -531,7 +531,7 @@ coreos:
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/etcd/server-key.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/etcd/server-key.pem to be written' && sleep 1; done"
       ExecStart=/usr/bin/etcd2 --advertise-client-urls=https://0.0.0.0:2379 \
                                --data-dir=/etc/kubernetes/data/etcd/ \
-                               --initial-advertise-peer-urls=https://0.0.0.0:2380 \
+                               --initial-advertise-peer-urls=https://127.0.0.1:2380 \
```

Comment:
The problem is that etcd has to listen to its domain, not on local host. It has to be available via ingress to calico within the worker nodes as well. This change is even more backwards. AFAIU the recent changes towards making etcd listening locally have just been wrong. Check https://github.com/giantswarm/k8s-vm on how it should be.